phoenix-core/src/it/java/org/apache/phoenix/end2end/ReverseScanIT.java

```diff
@@ -52,11 +52,11 @@
 import com.google.common.collect.Maps;
 
 
-public class ReverseScanIT extends BaseHBaseManagedTimeIT {
-    protected static final String ATABLE_INDEX_NAME = "ATABLE_IDX";
+public class ReverseScanIT extends BaseHBaseManagedTimeTableReuseIT {
+    protected static final String ATABLE_INDEX_NAME = generateRandomString();
```

Comment:
The only test case that uses the index is testReverseScanIndex. Please move this there.